### PR TITLE
Update Continuous Testing global configuration docs

### PR DESCRIPTION
**Pull request by peculiarism**
### What does this PR do?
Updating Continuous Testing documentation with new flags.

### Motivation
Some feedback received on the current docs

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 1]
```diff
@@ -85,11 +87,20 @@ In the global configuration file, you can configure the following options:
...
 
+`failOnCriticalErrors`
+: A boolean flag that will fail the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.
+
+`failOnMissingTests`
+: A boolean flag that will fail the CI job if at least one test is missing within a run (for example, if it has been removed/deleted). The default is set to `false`.
+
+`failOnTimeout`
+: A boolean flag that will fail the CI job if at least one test exceees the default test timeout. The default is set to `true`.
+
```
Comment (lefebvree) [inline content/en/continuous_testing/cicd_integrations/configuration.md:98]: We can add `pollingTimeout` in here as well if it helps making it more evident (we support both `config.pollingTimeout` & `config.global.pollingTimeout`)

**Review by lefebvree — APPROVED** [review]
LGTM

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 2]
```diff
@@ -70,10 +70,12 @@ To setup your client, Datadog API and application keys need to be configured. Th
     datadog-ci synthetics run-tests --apiKey "<API_KEY>" --appKey "<APPLICATION_KEY>"
     ```
 
-3. Or defined in a global configuration file:
+3. Or defined in a [global configuration fil](#global-configuration-file-options):
```
Comment (alai97) [inline content/en/continuous_testing/cicd_integrations/configuration.md:73]:
```suggestion
3. Or defined in a [global configuration file](#global-configuration-file-options):
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 3]
```diff
@@ -85,11 +87,24 @@ In the global configuration file, you can configure the following options:
 `datadogSite`
 : The Datadog instance to which request is sent. The default is `datadoghq.com`. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.
 
+`failOnCriticalErrors`
+: A boolean flag that will fail the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.
```
Comment (alai97) [inline content/en/continuous_testing/cicd_integrations/configuration.md:91]:
```suggestion
: A boolean flag that fails the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 4]
```diff
@@ -85,11 +87,24 @@ In the global configuration file, you can configure the following options:
...
+
+`failOnMissingTests`
+: A boolean flag that will fail the CI job if at least one test is missing within a run (for example, if it has been removed/deleted). The default is set to `false`.
```
Comment (alai97) [inline content/en/continuous_testing/cicd_integrations/configuration.md:94]:
```suggestion
: A boolean flag that fails the CI job if at least one test is missing in a run (for example, if it has been removed or deleted). The default is set to `false`.
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 5]
```diff
@@ -85,11 +87,24 @@ In the global configuration file, you can configure the following options:
...
+
+`failOnTimeout`
+: A boolean flag that will fail the CI job if at least one test exceees the default test timeout. The default is set to `true`.
```
Comment (alai97) [inline content/en/continuous_testing/cicd_integrations/configuration.md:97]:
```suggestion
: A boolean flag that fails the CI job if at least one test exceeds the default test timeout. The default is set to `true`.
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 6]
```diff
@@ -85,11 +87,24 @@ In the global configuration file, you can configure the following options:
...
+
 `files`
 : Glob pattern to detect Synthetic tests config files.
 
 `global`
-: Overrides of Synthetic tests applied to all tests ([see below for descriptions of each field](#configure-tests)).
+: Overrides of Synthetic tests applied to all tests ([see below for descriptions of each field](#additional-configuration)).
```
Comment (alai97) [inline content/en/continuous_testing/cicd_integrations/configuration.md:103]: Which header does this anchor link to?

```suggestion
: Overrides of Synthetic tests applied to all tests. [See below for descriptions of each field]().
```

File: content/en/continuous_testing/troubleshooting.md [hunk 7]
```diff
@@ -16,11 +16,14 @@ This page provides information to help you troubleshoot issues with Continuous T
 
 ## Terminology
 
-CI batch
+CI batch 
 : The group of Continuous Testing tests triggered through a continuous integration or continuous delivery (CI/CD) pipeline or the [Datadog Synthetic Monitoring API][2].
 
 Test run
-: A single run of a Continuous Testing test. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.
+: A single run of a Continuous Testing test. This can be [API][7] or [browser tests][8]. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.
```
Comment (alai97) [inline content/en/continuous_testing/troubleshooting.md:23]:
```suggestion
: A single run of a Continuous Testing test, which can be an [API][7] or [browser test][8]. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.
```

File: content/en/continuous_testing/troubleshooting.md [hunk 8]
```diff
@@ -16,11 +16,14 @@ This page provides information to help you troubleshoot issues with Continuous T
...
+
+Parallel test
+: A test that is run at the same time as another test. Parallel tests are part of the Continuous Testing product. In order to control how many tests you would like to run in parallel, configure parallelization in the [settings page][9].
```
Comment (alai97) [inline content/en/continuous_testing/troubleshooting.md:26]: Trying to rephrase "Parallel tests are part of the Continuous Testing product" in the first sentence.

```suggestion
: A Continuous Testing test that is run at the same time as another Continuous Testing test. To set how many tests you would like to run in parallel, configure parallelization on the [Continuous Testing Settings page][9].
```

File: content/en/continuous_testing/troubleshooting.md [hunk 9]
```diff
@@ -34,6 +37,12 @@ Execution rule
 
 Check whether you are using API endpoints to trigger your CI/CD test runs. In order to populate the CI Results Explorer with CI metadata, you must use one of Datadog's [native integrations][5], or the [NPM package][6].
 
+## Within your CI/CD pipeline
+
+### My tests are timing out in my CI. What do I do?
```
Comment (alai97) [inline content/en/continuous_testing/troubleshooting.md:42]:
```suggestion
### My tests are timing out in my CI pipeline
```

File: content/en/continuous_testing/troubleshooting.md [hunk 10]
```diff
@@ -34,6 +37,12 @@ Execution rule
...
+
+The first thing to check is which failure mode flags you are passing in your [global configuration file][10]. For CI runs that contain multiple tests, some tests will be queued based on the parallelization setting defined in your Continuous Testing [settings page][9]. You may need to adapt both your configuration and your parallelization based on your organisational needs.
```
Comment (alai97) [inline content/en/continuous_testing/troubleshooting.md:44]:
```suggestion
The first thing to check is which failure mode flags you are passing in your [global configuration file][10]. For CI runs that contain multiple tests, some tests are queued based on the parallelization setting defined on the [Continuous Testing Settings page][9]. You may need to adapt both your configuration and parallelization based on your organizational needs.
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some suggestions!

File: content/en/continuous_testing/cicd_integrations/configuration.md (hunk 6 above)
Comment (peculiarism) [reply content/en/continuous_testing/cicd_integrations/configuration.md:103]: @alai97 Do we want to remove the link to the `additional configuration` section?

**Review by alai97 — APPROVED** [review]

**Pull request merged**